src/test/org/apache/sqoop/importjob/configuration/MysqlImportJobTestConfiguration.java

```diff
@@ -65,4 +66,21 @@
   public String toString() {
     return getClass().getSimpleName();
   }
+
+  @Override
+  public Object[] getExpectedResultsForHive() {
```

Comment:
Source of code duplication again!
I would move these expected values into a helper class or a super class (choice is up to you).

Reply:
Hi Attila, 

I'd like to leave these as is.

I'd argue that these don't actually belong together, so it's not really code duplication but rather an accidental match.

I added a new column to the sample data of the mysql configuration to cover an edge case. The expected result in Hive changed for this configuration, and, on the other hand this edge case does not make sense to be added to the SQL server tests.

Reply:
Hi @fszabo2 ,

If that's the case:
Could you please at least add some information about the intention behind the values (A.K.A. explanatory comments by Uncle Bob), b/c right now for example if this test would fail I would not have much clue about what values do we have here, and why.

I'm thinking about something like:
this value is an int, XXX precision, so it should be YYYYY in Parquet/Hive

or something like
this value represents the biggest the longest possible values stored as type ZZZ, thus it contains N digits, and should be KKKK in Hive.

Could we add these information, thus put these "Magic numbers" into a context?

Reply:
Fair point!

I added a couple of comments, hope it clarifies the magic numbers.